buildSrc/src/main/kotlin/dev.openfeature.provider-conventions.gradle.kts

```diff
@@ -0,0 +1,90 @@
+import com.vanniktech.maven.publish.JavadocJar
+import com.vanniktech.maven.publish.KotlinMultiplatform
+
+plugins {
+    id("org.jetbrains.kotlin.multiplatform")
+    id("org.jetbrains.kotlinx.binary-compatibility-validator")
+    id("org.jlleitschuh.gradle.ktlint")
+    id("com.vanniktech.maven.publish")
+    id("org.jetbrains.dokka")
+}
+
+// Configure Dokka for documentation
+dokka {
+    dokkaPublications.html {
+        suppressInheritedMembers.set(true)
+        failOnWarning.set(true)
+    }
+    dokkaSourceSets.commonMain {
+        sourceLink {
+            localDirectory.set(file("src/"))
+            remoteUrl("https://github.com/open-feature/kotlin-sdk/tree/main/kotlin-sdk/src")
+            remoteLineSuffix.set("#L")
+        }
+    }
+}
+
+mavenPublishing {
+    configure(
+        KotlinMultiplatform(
+            javadocJar = JavadocJar.Dokka("dokkaGeneratePublicationHtml"),
+            sourcesJar = true,
+            androidVariantsToPublish = listOf("release")
+        )
+    )
+    signAllPublications()
+    coordinates(
+        groupId = "dev.openfeature.kotlin.contrib.providers",
+        version = findProperty("version").toString()
+    )
+    pom {
+        name.set("OpenFeature Kotlin SDK")
+        description.set(
+            "This is the Kotlin implementation of OpenFeature, a vendor-agnostic abstraction library for evaluating feature flags."
+        )
```

Comment:
Should we change that? 
It seems more related to the SDK than the contrib repository.